config/routes.rb

```diff
@@ -1,11 +1,12 @@
 Rails.application.routes.draw do
   root 'users#index'
 
-  # Posts routes
-  get '/users/:user_id/posts/:id', to: 'posts#index', as: 'user_post'
-  get '/users/:user_id/posts', to: 'posts#show', as: 'user_posts'
-
-  # Users routes
-  get "/users/:user_id/posts/:id", to: "users#show", as: "user_post_show"
-  get "/users/:user_id/posts", to: "users#index", as: "user_posts_index"
-end
\ No newline at end of file
+  get '/users/:id', to: 'users#show', as: 'user'
+  get '/users', to: 'users#index', as: 'users'
+  post '/users/:user_id/posts/:id/newcomment', to: 'comments#create', as: 'create_new_comment'
```

Comment:
- [ ] After refactoring your code above to app/views/comments/new.html.erb, please don't forget to add the routes here :+1: